bundles/io/org.eclipse.smarthome.io.rest/src/main/java/org/eclipse/smarthome/io/rest/JSONResponse.java

```diff
@@ -142,11 +145,18 @@ public static Response createResponse(Response.Status status, Object entity, Str
      */
     @Provider
     public static class ExceptionMapper implements javax.ws.rs.ext.ExceptionMapper<Exception> {
+
+        private final static Logger logger = LoggerFactory.getLogger(ExceptionMapper.class);
```

Comment:
I knew that you would mention this. :-)
But having seen that @bhuism struggled a bit creating this first PR, I didn't want to challenge him with amending commits, squashing, force pushing etc. just because of the "static" keyword here. Feel free to create a follow-up PR to remove it.